test/integration/integration_suite_test.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"k8s.io/gengo/args"
+	"k8s.io/kube-openapi/pkg/generators"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+)
+
+var package_name = "v1alpha1"
+var struct_name = "Model"
+
+var output_package = "main"
+var generated_base_name = "openapi_generated"
+var generated_file_name = generated_base_name + ".go"
+
+func TestGenerators(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Generators Suite")
+}
+
+var _ = Describe("Open API Definitions Generation", func() {
+
```

Comment:
I'll never understand why you keep an empty line at the beginning of every function :man_shrugging:

Reply:
It's against our style in theory, though not enforced as much as I'd prefer.